code/game/objects/items/food/sandwichtoast.dm

```diff
@@ -265,7 +265,7 @@
 	AddComponent(/datum/component/edible, check_liked = CALLBACK(src, PROC_REF(check_liked)))
 
 ///Eat it right, or you die.
```

Comment:
Can we improve this doc comment here since you are touching the file. This is not a good doc